src/Apps/gEvGen.cxx

```diff
@@ -87,6 +91,11 @@
               scheme for the generated kinematics of individual processes can
               still be in effect if enabled..
               ** Only use that option if you understand what it means **
+           --force
+              Forces the interaction of all injected events.
```

Comment:
I'm not loving the use of "injected events" here.  The term "event" is too overloaded, but to me it generally means an 'interaction'.  Perhaps just "Forces the interaction of all flux rays"

I'm also not clear what it means "if a neutrino flux is specified".  Doesn't one always need to specify some kind of flux?  Do you mean for all cases other than mono-energetic (in which case the mono flux acts as if force was alway in play)?

Reply:
I have changed the first comment accordingly.

Regarding the second comment. You are right, for monoenergetic it is always forced.